tests/test_json_to_parquet.py

```diff
@@ -517,6 +541,28 @@ def glue_crawler_role(namespace):
     role_name = f"{namespace}-pytest-crawler-role"
     glue_service_policy_arn = "arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole"
     s3_read_policy_arn = "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess"
+
+    # Cleanup if the role/policy already exist
```

Comment:
Oh, were we running into role/policy duplicates/conflicts here when trying to create role?

Reply:
Exactly! This is to make sure we're always running the test from a known state.